### ISSUE TITLE
Accessor for `pd.Series`

### ISSUE THREAD

**Issue by kwinkunks** [scienxlab/redflag#56]
As well as the DataFrame accessor, see #36 

- Check `is_continuous()`
- For continuous:
  - Basic stats like `df.describe()`
  - Outliers
  - Distribution (best fit)
  - Polynomial params to fit normal (Will need new function.)
  - `is_correlated()`
  - Possible to do something with `sklearn.dummy.DummyRegressor`? (Will need new function.)
- For categorical:
  - List ordered uniques w supports (Will need new function.)
  - Imbalance metrics
  - Pattern detection, eg sorted or repeating. Some kind of entropy? Normalized Markov transition matrix? (Will need new function.)
  - Result of `sklearn.dummy.DummyClassifier` using fake `X`. (Will need new function.)
